standard/ieee/802.3/draft/Comment blurb.txt

```diff
@@ -0,0 +1,21 @@
+The 802.3 Working Group follows the comment submission process outlined below. Individuals are invited to provide comments against the code provided at this GitHub site using this process.
+
+Comments must be categorized as Editorial or Technical.  
+
+Comments must clearly identify the location in the draft by page, Sub-clause, and line which is being commented against. 
+When commenting against the GitHub code directly the module name should be listed in the Sub-clause field and page may be left blank.
+
+Comments must clearly identify the issue which prompted the comment, specifically what is problematic in the draft that needs correction.  
+Comments must clearly identify a specific proposed change, typically by suggesting explicit text needed to resolve the comment. 
+Comments that do not include a clearly identify a proposed change may be rejected by the task force.  
+The ��Must be Satisfied�� field should be filled with ��No��; it is not used by the Task Force.  

```

Comment:
Please remove these binary characters.  Files need to be encoded as straight text files.